server/services/store/storetests/workspaces.go

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package storetests
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/mattermost/focalboard/server/model"
+	"github.com/mattermost/focalboard/server/utils"
+
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/mattermost/focalboard/server/services/store"
+)
+
+func StoreTestWorkspaceStore(t *testing.T, setup func(t *testing.T) (store.Store, func())) {
+	t.Run("UpsertWorkspaceSignupToken", func(t *testing.T) {
+		store, tearDown := setup(t)
+		defer tearDown()
+		testUpsertWorkspaceSignupToken(t, store)
+	})
+
+	t.Run("UpsertWorkspaceSettings", func(t *testing.T) {
+		store, tearDown := setup(t)
+		defer tearDown()
+		testUpsertWorkspaceSettings(t, store)
+	})
+
+	t.Run("GetWorkspaceCount", func(t *testing.T) {
+		store, tearDown := setup(t)
+		defer tearDown()
+		testGetWorkspaceCount(t, store)
+	})
+}
+
+func testUpsertWorkspaceSignupToken(t *testing.T, store store.Store) {
+	t.Run("Insert and update workspace with signup token", func(t *testing.T) {
+		workspaceID := "0"
+		workspace := &model.Workspace{
+			ID:          workspaceID,
+			SignupToken: utils.CreateGUID(),
+		}
+
+		// insert
+		err := store.UpsertWorkspaceSignupToken(*workspace)
+		require.Nil(t, err)
```

Comment:
Minor thing... could you use `require.NoError` for these cases.  It adds some extra type checking for the test.